scripts/generate/templates/checkout/files/extension.config.yml

```diff
@@ -1,6 +1,7 @@
 ---
 extension_points:
   - Checkout::Feature::Render
+extension_title: App Feature
```

Comment:
I feel like this should just be `title`/ `name`. It's hard to remove the `extension_` from `extension_points`, but I don't think we should copy that `extension_` prefix to all the other fields just for consistency.

Reply:
Also, nit: can we put this at the top? The name of a thing goes near the top :)

Reply:
> I don't think we should copy that extension_ prefix to all the other fields just for consistency.

Makes sense. I was more worried that it would be confusing for partners which title we are talking about. But I'll change that.